tests/test_mpi.py

```diff
@@ -12,7 +13,7 @@
 from devito.mpi import MPI
 from examples.seismic.acoustic import acoustic_setup
 
-pytestmark = skipif(['yask', 'ops', 'nompi'])
+pytestmark = skipif(['yask', 'ops', 'nompi'], whole_module=True)
```

Comment:
can you remind me what the `whole_module` is for?

Reply:
This skips the entire file without reading it. Without it,  `pytest` collects and initializes the classes then skip the tests only, so this avoids initialization of TestAcoustic for the non-mpi builds.